spec/inputs/elasticsearch_spec.rb

```diff
@@ -120,40 +123,35 @@
     end
 
     before(:each) do
-      client = Elasticsearch::Client.new
-      expect(Elasticsearch::Client).to receive(:new).with(any_args).and_return(client)
+      expect(Elasticsearch::Client).to receive(:new).with(any_args).and_return client = Elasticsearch::Client.new
```

Comment:
I think the version in 2 lines is more readable, unless for some reason this line is mandatory to fix the problem

Reply:
```suggestion
      client = Elasticsearch::Client.new 
      expect(Elasticsearch::Client).to receive(:new).with(any_args).and_return(client)
```